lib/versioning/composer/index.ts

```diff
@@ -57,7 +57,7 @@ function normalizeVersion(input: string): string {
   return convertStabilityModifier(output);
 }
 
-function composer2npm(input: string): string {
+function composer2npm(input: string, allowPatchVersion = false): string {
```

Comment:
Under which scenarios would we want this to be false?

Reply:
@rarkins 
when this is set to true it will return 2.3.6 which indicates that it is stable but when installing it should return 2.3.6-p1. Sot this only has to be set to true for validating if it is a stable release because the original npm package semver will return that 2.3.6-p1 is not stable so we will check against 2.3.6 when the original value is 2.3.6-p1

Reply:
Could we handle it in the `isStable()` function instead? e.g.

```
const isStable = (version: string): boolean =>
  version && npm.isStable(composer2npm(version.replace(/^-(p|patch)+[0-9]+$/, ''));
```